### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by tpayet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# MeiliSearch SDKs
+# MeiliSearch integrations and sdks guide
```
Comment (curquiza) [inline README.md:1]:
```suggestion
# MeiliSearch integrations and SDKs guide
```

File: README.md [hunk 2]
```diff
@@ -11,3 +11,20 @@ Fully maintained:
 Work in progress:
 
 - [Java](https://github.com/meilisearch/meilisearch-java)
+
```
Comment (curquiza) [inline README.md:14]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -11,3 +11,20 @@ Fully maintained:
...
+
+This repository aims to centralize all informations about the differents sdks & integrations that are built around MeiliSearch.
```
Comment (curquiza) [inline README.md:16]:
```suggestion
This repository aims to centralize all information about the different SDKs & integrations that are built around MeiliSearch.
```

File: README.md [hunk 4]
```diff
@@ -11,3 +11,20 @@ Fully maintained:
...
+
+## Documentation
+
+MeiliSearch documentations is at [docs.meilisearch.com](https://docs.meilisearch.com).
```
Comment (curquiza) [inline README.md:20]:
```suggestion
Here is the [MeiliSearch documentation](https://docs.meilisearch.com).
```

File: README.md [hunk 5]
```diff
@@ -11,3 +11,20 @@ Fully maintained:
...
+There is no documentation for the different sdks except the dedicated READMEs. There is a [discussion about it](https://github.com/meilisearch/sdks/issues/3).
```
Comment (curquiza) [inline README.md:21]:
```suggestion
There is no documentation for the different SDKs except the dedicated READMEs. There is a [discussion about it](https://github.com/meilisearch/sdks/issues/3).
```

File: README.md [hunk 6]
```diff
@@ -11,3 +11,20 @@ Fully maintained:
...
+
+## Building an integration
+
+If you want to build an integration for MeiliSearch, you are more than welcome to and we would be happy to help you :)
+Here are some guidelines that you should follow though, if we want to garantee consistency across the differents integrations and technical quality for the different MeiliSearch users.
```
Comment (curquiza) [inline README.md:26]:
```suggestion
Here are some guidelines that you should follow though, if we want to guarantee consistency across the differents integrations and technical quality for the different MeiliSearch users.
```

File: README.md [hunk 7]
```diff
@@ -11,3 +11,20 @@ Fully maintained:
...
+
+- About testing: Currently, the most tested sdk is the Ruby one but if you want to start writing test you can read what the PHP sdk is testing to get an overview of minimum viable tests that your sdk should support.
```
Comment (curquiza) [inline README.md:28]:
```suggestion
#### About testing

If you want to start writing tests you can read what the PHP SDK is testing to get an overview of minimum viable tests that your SDK should support. The JS and the Ruby SDKs are also good examples of deeper tests.
```

File: README.md [hunk 8]
```diff
@@ -11,3 +11,20 @@ Fully maintained:
...
+
+- README: All the README should follow the same patterns. This reository should hold the README template and at some point we might want to automaticaly update all the readmes from here.
```
Comment (curquiza) [inline README.md:30]:
```suggestion
#### README

All the README should follow the same patterns. This repository should hold the README template and at some point, we might want to automatically update all the readmes from here. 
```

**Comment by curquiza** [comment]
Can you add the Swift repo in your PR? 🙂

File: README.md [hunk 9]
```diff
@@ -11,3 +11,20 @@ Fully maintained:
...
+This repository aims to centralize all informations about the differents sdks & integrations that are built around MeiliSearch.
+
+## Documentation
+
+MeiliSearch documentations is at [docs.meilisearch.com](https://docs.meilisearch.com).
+There is no documentation for the different sdks except the dedicated READMEs. There is a [discussion about it](https://github.com/meilisearch/sdks/issues/3).
+
+## Building an integration
+
+If you want to build an integration for MeiliSearch, you are more than welcome to and we would be happy to help you :)
```
Comment (curquiza) [inline README.md:25]:
```suggestion
If you want to build an integration for MeiliSearch, you are more than welcome to and we would be happy to help you :)

```

**Comment by tpayet** [comment]
Sorry for all of these silly typos 😅

**Comment by curquiza** [comment]
No prob' mister 🙂
LGTM!

**Pull request merged**